client/blocks/plugins-update-manager/context/index.tsx

```diff
@@ -1,24 +1,39 @@
-import React, { createContext, ReactNode } from 'react';
+import React, { createContext, ReactNode, useState } from 'react';
 import type { SiteSlug } from 'calypso/types';
 
 interface PluginUpdateManagerContextProps {
 	siteSlug: SiteSlug;
 	isEligibleForFeature: boolean;
 }
 
-const PluginUpdateManagerContext = createContext< PluginUpdateManagerContextProps >( {
+interface PluginUpdateManagerContextState {
+	hasUserManagedPlugins: boolean;
+	setHasUserManagedPlugins: ( hasUserManagedPlugins: boolean ) => void;
```

Comment:
```suggestion
	siteHasManagedPlugins: boolean;
	setSiteHasManagedPlugins: ( siteHasManagedPlugins: boolean ) => void;
```

I've not found any WP documentation on this, but I was taught to use statement variable naming rather than questioning since they're supposed to contain the answer. So `pluginIsManaged(!)` rather than `isPluginManaged(?)` if that makes sense?

What do you think about making the naming about the site that these plugins are installed rather than the user that's looking at the scheduled updates page?

Reply:
I like the idea! I'll refactor.

Reply:
Renamed to `siteHasEligblePlugins`, since I think it sounds more clear and less negative than `siteHasUnmanagedPlugins`